src/Maker/MakeResetPassword.php

```diff
@@ -371,8 +371,14 @@ private function generateRequestEntity(Generator $generator, ClassNameDetails $r
 
         $generator->writeChanges();
 
+        $useAttributesForDoctrineMapping = $this->doctrineHelper->isDoctrineSupportingAttributes() && $this->doctrineHelper->doesClassUsesAttributes($requestClassNameDetails->getFullName()) && $this->doctrineHelper->canUseAttributes();
```

Comment:
As mentioned earlier, I think the `canUseAttributes()` is totally redundant and cab be removed: `isDoctrineSupportingAttributes()` covers us.

Reply:
done